kyuubi-server/src/main/scala/org/apache/kyuubi/server/BackendServiceMetric.scala

```diff
@@ -197,6 +217,24 @@ trait BackendServiceMetric extends BackendService {
             case t: TI16Column => t.getValues.size()
             case t: TBoolColumn => t.getValues.size()
             case t: TByteColumn => t.getValues.size()
+            case t: TBinaryColumn
+                if resultFormat == "arrow" =>
+              t.getValues.asScala.map(listBuff => {
+                val batchBytes = listBuff.array()
+                val input = new ByteArrayInputStream(batchBytes)
+                val readChannel = new ReadChannel(Channels.newChannel(input))
```

Comment:
The CPU/memory overhead looks high, it's dangerous do such thing on Kyuubi server